sensor/common/registry/registry_store.go

```diff
@@ -89,12 +86,27 @@ func (rs *Store) UpsertRegistry(ctx context.Context, namespace, registry string,
 				Insecure: !secure,
 			},
 		},
-	})
+	}
+}
+
+// UpsertRegistry upserts the given registry with the given credentials in the given namespace into the store.
+func (rs *Store) UpsertRegistry(ctx context.Context, namespace, registry string, dce config.DockerConfigEntry) error {
+	secure, err := rs.checkTLS(ctx, registry)
+	if err != nil {
+		return errors.Wrapf(err, "unable to check TLS for registry %q", registry)
+	}
+
+	regs := rs.getRegistries(namespace)
+
+	// remove http/https prefixes from registry, matching may fail otherwise, the created registry.url will have
+	// the appropriate prefix
+	registry = urlfmt.TrimHTTPPrefixes(registry)
+	err = regs.UpdateImageIntegration(createImageIntegration(registry, dce, secure))
 	if err != nil {
 		return errors.Wrapf(err, "updating registry store with registry %q", registry)
 	}
 
-	log.Debugf("Upserted registry %q for namespace %q into store", registry, namespace)
+	log.Debugf("upserted registry %q for namespace %q into store", registry, namespace)
```

Comment:
I think for logging, uppercase is fine

Reply:
interesting, ok i'll change it back, old habit I suppose. (in my past if a message didn't end in with a `.` or started with a proper noun, then lowercase was preferred) - throughout the codebase I don't see a convention one way or the other, some upper, some lower.